common/locales/en/character.json

```diff
@@ -60,8 +60,11 @@
     "gearAchievement": "You have earned the \"Ultimate Gear\" Achievement for upgrading to the maximum gear set for a class! You have attained the following complete sets:",
     "moreGearAchievements": "To attain more Ultimate Gear badges, change classes on <a href='/#/options/profile/stats' target='_blank'>your stats page</a> and buy up your new class's gear!",
     "armoireUnlocked": "You've also unlocked the <strong>Enchanted Armoire!</strong> Click on the Enchanted Armoire Reward for a random chance at special Equipment! It may also give you random XP or food items.",
-    "ultimGearName": "Ultimate Gear",
-    "ultimGearText": "Has upgraded to the maximum weapon and armor set for the following classes:",
+    "ultimGearName": "Ultimate Gear - <%= ultClass %>",
+    "warriorUltimGearText": "Has upgraded to the maximum weapon and armor set for the Warrior class.",
```

Comment:
This should probably be 

```json
"ultimGearText": "Has upgraded to the maximum weapon and armor set for the <%= class(locale) %> class.",
```

And you'd pass in `i18n.t('warrior')` as the parameter:

```js
let text = i18n.t('ultimGearText', { class: i18n.t(data.path) }, data.language);
```

Assuming `data.path` is the class.